doc/nrf/installation/install_ncs.rst

```diff
@@ -48,6 +48,8 @@ Depending on your preferred development environment, install the following requi
 
       * The latest version of the :ref:`requirements_clt` package.
         Download it from the `nRF Command Line Tools`_ page.
+      * The latest version of :ref:`SEGGER J-Link <requirements_jlink>`.
```

Comment:
Is this actually the latest versions? Isn't just the latest approved version?  (currently 7.96s?) If that's the case, we should have this (and every other instance in the docs) as a shortcut.txt entry and modify this for each release

Reply:
Sure, the shortcut solution is the second option on my list, but note the discussion in https://nordicsemi.atlassian.net/browse/NRFU-1160

Reply:
Applied the shortcut for J-Link version.